unit_tests/src/test_host_list_context_string.cpp

```diff
@@ -0,0 +1,181 @@
+//
+// The following unit tests were implemented based on code examples from
+// "cppreference.com". This code is licensed under the following:
+//
+//   - Creative Commons Attribution-Sharealike 3.0 Unported License (CC-BY-SA)
+//   - GNU Free Documentation License (GFDL)
+//
+// For more information about these licenses, visit:
+//
+//   - https://en.cppreference.com/w/Cppreference:FAQ
+//   - https://en.cppreference.com/w/Cppreference:Copyright/CC-BY-SA
+//   - https://en.cppreference.com/w/Cppreference:Copyright/GDFL
+//
+
+#include <catch2/catch.hpp>
+
+#include <string>
+#include <sstream>
+#include <vector>
+#include <tuple>
+#include <optional>
+
+#include <irods/irods_plugin_base.hpp>
```

Comment:
Use quotes instead of angle brackets for headers that are in the same repository.

Reply:
done